### ISSUE TITLE
Vote for which file transfer tools

### ISSUE THREAD

**Issue by richelbilderbeek** [UPPMAX/naiss_file_transfer_course#2]
There are many tools for file transfer, such as (incomplete list) scp, sftp, rsync, FileZilla, WinSCP, Cyberduck, etc.

The options suggested were:

  - scp 
  - sftp 
  - rsync 
  - Gui based tools 
      - Filezilla 
      - Winscp 
      - Cyberduck 

Which ones will this course use? All?

**Comment by richelbilderbeek** [comment]
We follow a connectivist's approach here:

- FileZilla first, as this is close to the experience of most learners, assuming most learners use a desktop environment
- rsync second, as this seems to be one of the best and flexible tools, the only one (?) for sensitive data transfer between NAISS HPC centers

**Comment by bclaremar** [comment]
Thanx for interesting input!

**Comment by richelbilderbeek** [comment]
Votes till now:

- FileZilla: Björn, Richel
- rsync: Björn, Richel
- scp: Viktor
- WinSCP, if better for Windows users: Viktor
- Transferring webservices: Viktor
- Rclone: Viktor

**Comment by bclaremar** [comment]
@VRehnberg 
about the Filezilla adware. 
This is from the Alvis side?
Local user client should not matter in this case?

**Comment by VRehnberg** [comment]
No, it was bundled with the FileZilla client (server side wouldn't have been as bad). I'm not sure if they still do this, but googling it is clear that they've done this in the past and to some degree also tried to obscure this inclusion.

**Comment by richelbilderbeek** [comment]
For me, this does not change my vote: it is still the best FOSS cross-platform user-friendly GUI tool around :-) . Sure, if there is a better one, I'd be happy to hear it :+1:

**Comment by bclaremar** [comment]
No malware at least? Security issues?

I think we can use it when we demo.
And inform about the C3SE policy in an info box

**Comment by richelbilderbeek** [comment]
I think the bundled adware is gone: I've found a FileZilla forum post [here](https://forum.filezilla-project.org/viewtopic.php?t=53629) where a download button is shows with the text 'this install may contain bundled offers) [image from that forum here](https://forum.filezilla-project.org/download/file.php?id=4136&sid=237cd5cad1a40492eed560f11a442aa3).

Checking as of today, there is no longer bundled things with the download:

![Screenshot from 2024-12-20 09-55-33](https://github.com/user-attachments/assets/e3807463-525a-432b-a607-12d6fc0e013e)

Hence, I feel FileZilla is fine, unless there is new evidence to prove the contrary.

**Comment by VRehnberg** [comment]
> No malware at least?

Not deliberately at least, as it should be possible to opt-out.

> Security issues?

I don't know.

> I think we can use it when we demo. And inform about the C3SE policy in an info box

Fine, by me. Just wanted to let you make an informed choice.

> Checking as of today, there is no longer bundled things with the download

You might want to check for the other operating systems as well. At least 3.67.1 (3.68.1 is latest) was bundling stuff for the Windows installation [source](https://www.youtube.com/watch?v=83R-IgCAvfQ). I don't have Windows available so I can't check how it is now.

**Comment by richelbilderbeek** [comment]
@VRehnberg : I love people that share their sources! Thanks!

So this is the only possible maybe-problematic dialog option:

![Screenshot from 2024-12-20 11-52-57](https://github.com/user-attachments/assets/6dcf8555-6a6e-4580-8c67-9afc41890d62)

I feel this is not evil nor a dark pattern and using FileZilla is one mouse click away from being perfect :+1:

**Comment by bclaremar** [comment]
closing
